docs/docs/glossary.rst

```diff
@@ -284,3 +284,12 @@ The verbalization involves verbalizing the task input fields for the input, and
    :alt: The unitxt prompt structure
    :width: 75%
    :align: center
+
+
+Serialization
+--------------
+
+Is the process of taking a specific field of a task a converting it into a string.
+This process is controlled by the :ref:`Serializer <serializers>`.
+Unlike the verbalization which is task specific, namely putting into words an input of a specific task, the serialization
```

Comment:
I'm not sure this is distinction is right (verbalization is task specific vs serialization which I not).

**Verbalization** is the process of taking the task fields and converting them into their
textual representation, which is provided as input to the model.

Type serialization is part of the verbalization process and is often related to the template used.
The other type is the template formatting, which verbalizes how the text representation of the fields are arranged in the prompt.